backend/fms_core/viewsets/_utils.py

```diff
@@ -356,6 +355,25 @@ def default_prefilling(sample_id: int, template, user_prefill_data):
                 for column_name, value in user_prefill_data.items():
                     sample_row_dict[column_name] = value
             return sample_row_dict
+        
+        def default_batch_prefilling(sample_id_list: List[int], template, batch_sheet_name, user_prefill_data):
+            # Get the list of attributes to group by (prefill[0] = sheet_name, prefill[2] = queryset_name)
```

Comment:
what's `prefill[1]`?

Reply:
prefill[1] is the template column name ... prefill[3] is property name and prefill[4] will now be a parsing function to extract a value out of an attribute content ...